pitest-maven-verification/src/test/java/org/pitest/PitMojoIT.java

```diff
@@ -149,13 +157,15 @@ public void shouldCorrectlyTargetTestsWhenMultipleBlocksIncludeALine() throws Ex
    */
   @Test
   public void shouldSkipSiteReportGeneration() throws Exception {
-    File testDir = prepare("/pit-site-skip");
-
-    verifier.executeGoals(Arrays.asList("test", "org.pitest:pitest-maven:mutationCoverage", "site"));
-
+    File testDir = prepareSiteTest("/pit-site-skip");
     File siteParentDir = buildFile(testDir, "target", "site");
+    
+    verifier.executeGoal("site");
+
+    assertThat(siteParentDir.exists()).isEqualTo(true);
     assertThat(buildFile(siteParentDir, "pit-reports").exists()).isEqualTo(false);
```

Comment:
`assertThat` has a FileAssert-extension, so it is sufficient to write `assertThat(...).exists()`

same with `assertThat(...).isTrue()`